### DISCUSSION TITLE
Is There A kernel version Switcher For Nissa the board name?/

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1285] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is There A kernel version Switcher For Nissa the board name?/

**Comment by schoolexploitkid** [comment, marked as answer]
yes, download the nissa shim from darkn mirror here: https://dl.darkn.bio/api/raw/?path=/SH1mmer/nissa.zip

**Reply by schoolexploitkid** [reply]
then run kvs code on it

**Reply by ghost** [reply]
how do I Run kvs code on it ???
